client.go

```diff
@@ -24,7 +24,7 @@ type Logger interface {
 //  func init() {
 //   statsd.UDPPayloadSize = 16 * 1024
 //  }
-var UDPPayloadSize int = 512
+var UDPPayloadSize = 512
```

Comment:
A fan of fixing these lint issues. The only downside is that it's going to cause more of a divergence between our fork and the original repo.

Reply:
I'm (unfortunately) operating on the assumption that the upstream repo is unmaintained, and that we're on our own for now.  :cry: 

@Jonnymcc submitted a PR with some of the reconnect stuff in it, and it's going unaddressed, along with a slew of other PRs.

There was talk about switching to use [this client instead](https://github.com/alexcesaro/statsd), which has some attractive-looking benchmarks, but that also looks to be sporadically maintained.